io.mpv.Mpv.yml

```diff
@@ -499,7 +498,7 @@ modules:
         url: https://github.com/jpsdr/x264
         mirror-urls:
           - https://code.videolan.org/videolan/x264.git
-        commit: c24e06c2e184345ceb33eb20a15d1024d9fd3497
+        commit: 3a8b5be2a256ffc6d8ec4fb35ecfb335d5a4a03b
```

Comment:
mirror is missing this commit. Can url and mirror-urls be switched over? Flathub will be fine using main repo.

BTW: it would be better to track  stable branch instead of master but this mirror doesn't have stable